benchmarks/perf-tool/okpt/test/steps/steps.py

```diff
@@ -278,21 +278,23 @@ def __init__(self, step_config: StepConfig):
         self.dataset = parse_dataset(dataset_format, dataset_path,
                                      Context.INDEX)
 
+        self.doc_count = parse_int_param('doc_count', step_config.config, {},
+                                         self.dataset.size())
+        self.doc_count = min(self.doc_count, self.dataset.size())
+
     def _action(self):
 
         def action(doc_id):
             return {'index': {'_index': self.index_name, '_id': doc_id}}
 
-        id = 0
         index_responses = []
-        while True:
+        for doc_id_ in range(0, self.doc_count, self.bulk_size):
```

Comment:
Was that intentional to have variable name end with '_'?

Reply:
Yes, I was getting a warning about overriding the doc_id on 287 and I saw somewhere that that is a convention for avoiding overrides. Do you have any other suggestions?

Reply:
Why not different variable name?

Reply:
Actually you are right, I should switch the var name. Originally, I was thinking that it was a doc_id so I should refer to it as that, but here it is not necessarily a doc id. Its the current place in the dataset. I will switch to i.